DI-435/find_vcfs_to_merge.py

```
@@ -335,6 +352,53 @@ def get_failed_samples(qc_status_df):
     return fail_sample_names
 
 
+def find_medicover_vcf_files(projects):
```

Ruff (0.8.0)

Missing return type annotation for public function `find_medicover_vcf_files`

(ANN201)